app/views/mailers/verification_mailer/account_ownership_verification_email.html.erb

```diff
@@ -0,0 +1,18 @@
+<% host = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}" %>
+<% verify_url = "#{host}/verify_email_ownership?confirmation_token=#{@user.email_authorizations.order('created_at DESC').first.confirmation_token}&username=#{@user.username}" %>
```

Comment:
you can use something like:

```erb
<%= app_url(verify_email_authorizations_path(confirmation_token: token, username: @user.username)) %>
```

where token comes from the email action as suggested by @Zhao-Andy